core/docker/build.sh

```diff
@@ -63,15 +63,19 @@ CONTAINER="trino:${TRINO_VERSION}"
 
 docker build "${WORK_DIR}" --pull --platform linux/amd64 -f amd64.dockerfile -t "${CONTAINER}-amd64" --build-arg "TRINO_VERSION=${TRINO_VERSION}"
```

Comment:
Let's rename these Dockerfiles to `server-` to avoid confusion

Reply:
Done!